chartpress.yaml

```diff
@@ -19,6 +19,7 @@ charts:
       - helm-chart
       - acceptance-tests
       - scripts/init-realm
+      - certificates
```

Comment:
No it should not. Because when stuff under `certificates` is updated, then this leads to a new certificates image - this is a part of the renku helm chart. Because the certificates image is specified in the `global` section of the renku helm chart.

The certificates library helm chart has no values. It uses whatever is specified under `global.certificates` in any of the other true helm charts that it operates in.